src/data/json_to_parq.py

```diff
@@ -48,15 +48,15 @@ def clean_for_parquet(df):
 
 def parquet_to_gcs(df, day):
```

Comment:
Add docstrings to these functions too.

Reply:
Added docstrings to all functions in `json_to_parq.py` file.